src/controllers/reporting.controller.ts

```diff
@@ -18,7 +18,26 @@ export const getZipAlertList = asyncHandler(
         req: Request<{}, {}, {}, {}>,
         res: Response,
         next: NextFunction
-    ) => {}
+    ) => {
+        //var arr: Number[] = []
+        var { zipList } = require('../config/queue.ts');
+        /*if(zipList != null){
```

Comment:
Can you remove the commented code here as well? I think this will help those reading the codebase like Dr. Bumgardner.